microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/AnnotationActivationCompatibilityTest.java

```diff
@@ -27,13 +27,14 @@
  * Just verify a raw token value can be injected into a variable in an application class a compare it has expected value
```

Comment:
Maybe I don't understand the wording, but perhaps you meant that "a compare" to be "and compare"? 
Just guessing. Anyway the class name is about activating the subsystem via annotations, which is fine, so maybe this is what should be mentioned in this line. The test method description, later, is covering the specific case requirements already, IMO.

Reply:
Thanks, will edit.